utils/cq/d_logging_check.py

```diff
@@ -115,12 +119,13 @@ def __next__(self):
             raise StopIteration from exc
 
 
-# Logging macros that expect a new-line.
+# Logging macros where the new-line is added if missing.
 PREFIXES = ['D_ERROR', 'D_WARN', 'D_INFO', 'D_NOTE', 'D_ALERT', 'D_CRIT', 'D_FATAT', 'D_EMIT',
             'D_TRACE_INFO', 'D_TRACE_NOTE', 'D_TRACE_WARN', 'D_TRACE_ERROR', 'D_TRACE_ALERT',
-            'D_TRACE_CRIT', 'D_TRACE_FATAL', 'D_TRACE_EMIT', 'RPC_TRACE', 'RPC_ERROR']
+            'D_TRACE_CRIT', 'D_TRACE_FATAL', 'D_TRACE_EMIT', 'RPC_TRACE', 'RPC_ERROR',
+            'VOS_TX_LOG_FAIL', 'VOS_TX_TRACE_FAIL', 'D_DEBUG']
 
-# Logging macros that do not expect a new-line.
+# Logging macros where a new-line is always added.
```

Comment:
Do we plan to eventually update these macros to have the same behavior with newlines?